src/App.tsx

```diff
@@ -1,20 +1,47 @@
-import { PeoplePage } from './components/PeoplePage';
 import { Navbar } from './components/Navbar';
 
 import './App.scss';
+import { Outlet } from 'react-router-dom';
+import { useContext, useEffect } from 'react';
+import { Person } from './types';
+import { getPeople } from './api';
+import { PeopleContext } from './contexts/PeopleContext';
+import { ErrorContext } from './contexts/ErrorContext';
```

Comment:
Ensure that the import path './contexts/ErrorContext' is correct. Previously, there was a misspelling in the directory name 'contexts'. Verify that this has been corrected.